pages/Admins/index.tsx

```diff
@@ -1,8 +1,28 @@
+import * as React from 'react';
 import Layout from "../../components/Layout/Layout";
+import styles from './Admins.module.css';
+import { Grid, Button, Box } from "@mui/material";
 
 const AdminPage: React.FunctionComponent = () => {
+
     return(
-        <Layout title='Admins'/>
+      <main>
+        <Layout title='Admins'>
+          <main className={styles.main}>
+            <Grid container className={styles.container}>
+              <Grid item xs={8}>
+                <h2 className={styles.h2}>ADMIN ACCOUNTS</h2>
+              </Grid>
+              <Grid item xs={2}>
+                <Button className={styles.button}>INVITE ADMIN</Button>
+              </Grid>
+              <Grid container className={styles.table}>
```

Comment:
nit: when applying CSS rules to certain elements/components access the rules within the style object like this instead: `styles['table']` this is for code consistency and also allows us to handle rules written in kebab case 
```suggestion
              <Grid container className={styles['table']}>
```